src/openfermion/measurements/_equality_constraint_projection_test.py

```diff
@@ -55,14 +54,14 @@ def setUp(self):
 
     def test_linearize_term(self):
         past_terms = set()
-        for term, coefficient in self.fermion_hamiltonian.terms.items():
+        for term, __ in self.fermion_hamiltonian.terms.items():
```

Comment:
Can we configure your tool to use just one underscore `_` instead of two?

Reply:
Done